x/ibc/applications/transfer/keeper/relay.go

```diff
@@ -144,7 +154,25 @@ func (k Keeper) SendTransfer(
 		timeoutTimestamp,
 	)
 
-	return k.channelKeeper.SendPacket(ctx, channelCap, packet)
+	if err := k.channelKeeper.SendPacket(ctx, channelCap, packet); err != nil {
+		return err
+	}
+
+	defer func() {
+		telemetry.SetGaugeWithLabels(
+			[]string{"tx", "msg", "ibc", "transfer"},
+			float32(token.Amount.Int64()),
+			[]metrics.Label{telemetry.NewLabel("denom", fullDenomPath)},
+		)
+
+		telemetry.IncrCounterWithLabels(
```

Comment:
I think it might make sense to just move this to the core `SendPacket` func. Having a label for the portID/channelID being sent from would indicate the module being used

Reply:
hmm how do you know if the chain is the source or sink from that? I think it's a valuable metric

Reply:
I think I just mean the non-transfer specific metrics here

Reply:
added